providers/azure.go

```diff
@@ -210,3 +270,8 @@ func (p *AzureProvider) GetEmailAddress(ctx context.Context, s *sessions.Session
 
 	return email, err
 }
+
+// ValidateSessionState validates the AccessToken
+func (p *AzureProvider) ValidateSessionState(ctx context.Context, s *sessions.SessionState) bool {
```

Comment:
Nice! Thanks for this addition.

Reply:
Credits must go to @m-wcislo who did the original PR :)